README.md

```diff
@@ -1 +1,50 @@
 # ros2_robotiq_gripper
+
+## Building
+
+#### Create ROS2 workspace
+```
+export COLCON_WS=~/workspace/
+mkdir -p $COLCON_WS/src
+
+```
+
+#### Pull relevant packages, install dependencies, compile, and source the workspace by using
+
+```
+cd $COLCON_WS
+git clone https://github.com/PickNikRobotics/ros2_robotiq_gripper.git src/ros2_robotiq_gripper.git
+vcs import src --skip-existing --input src/ros2_robotiq_gripper/ros2_robotiq_gripper.repos
+rosdep update
+rosdep install --ignore-src --from-paths src -y -r
+colcon build --cmake-args -DCMAKE_BUILD_TYPE=Release
+source install/setup.bash
+
+```
+
+## Launch
+
+#### With real hardware
+
+```
+ros2 launch robotiq_driver robotiq_control.launch.py
+
+```
+
+#### With ros2_control fake hardware
+
+```
+ros2 launch robotiq_driver robotiq_control.launch.py use_fake_hardware:=true
+```
+
+#### With Ignition simulation
+
+```
+ros2 launch robotiq_driver robotiq_simulation.launch.py
+```
```

Comment:
This looks fine in RViz, but not in Ignition:

https://user-images.githubusercontent.com/96085371/191117386-3f84c568-831b-45aa-b2e6-9bc65c6aed35.mp4

Reply:
@corycrean I would say the ignition is not installed properly